man/dockerd.8.md

```diff
@@ -289,11 +290,20 @@ unix://[/path/to/socket] to use.
   has no effect.
 
   This setting will also enable IPv6 forwarding if you have both
-  **--ip-forward=true** and **--fixed-cidr-v6** set. Note that this may reject
-  Router Advertisements and interfere with the host's existing IPv6
+  **--ip-forward=true** and an IPv6 enabled bridge network. Note that this
+  may reject Router Advertisements and interfere with the host's existing IPv6
   configuration. For more information, consult the documentation about
   "Advanced Networking - IPv6".
 
+**--ip-forward-no-drop**=**true**|**false**
+  When **false**, the default, if Docker enables IP forwarding itself (see
+  **--ip-forward**), and **--iptables** or **--ip6tables** are enabled, it
+  will also set the default policy for the FORWARD chain in the iptables
```

Comment:
Nit; avoid future tense; this could be "it also sets the ..."

Reply:
Done - there are a few existing examples of that in the text, I guess we should tidy those too (`--default-address-pool`, `--ip-forward`, `--ipv6`, `--userns-remap`, `zfs.fsname`).

Reply:
Ah, yes, we could make a pass at doing so. We also still need to change our packaging to use the one from this repository in stead of the copy that still lives in docker/cli 🙈

Reply:
Oh, right, there are two `docker.md.8`'s! So we should be keeping them in step for now?